guides/source/active_record_querying.md

```diff
@@ -557,6 +557,19 @@ In other words, this query can be generated by calling `where` with no argument,
 SELECT * FROM clients WHERE (clients.locked != 1)
 ```
 
+### OR Conditions
+
+`OR` condition between two relations can be build by calling `or` on the first relation
+and passing the second one as an argument.
+
+```ruby
+Client.where(locked: true).or(Client.where(orders_count: [1,2,5]))
```

Comment:
Generated SQL is `clients.orders_count IN (1,3,5)` but the condition is `Client.where(orders_count: [1,2,5])`.